dubbo-registry/dubbo-registry-nacos/src/main/java/com/alibaba/dubbo/registry/nacos/NacosRegistryFactory.java

```diff
@@ -23,7 +23,7 @@
 import com.alibaba.nacos.api.NacosFactory;
 import com.alibaba.nacos.api.exception.NacosException;
 import com.alibaba.nacos.api.naming.NamingService;
-import com.alibaba.nacos.client.naming.utils.StringUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Use StringUtils in dubbo will be better.

Reply:
+1, https://github.com/apache/dubbo/pull/7611